prov/efa/src/rxr/rxr_atomic.c

```diff
@@ -0,0 +1,477 @@
+#include <ofi_atomic.h>
```

Comment:
This file is missing a copyright header.

Reply:
added. Thank you!